.environment-scripts/install_nceplibs.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -e
+
+CLONE_PREFIX=$1
+INSTALL_PREFIX=$2
+PLATFORM=$3
+COMPILER=$4
+
+URL=https://github.com/NCAR/NCEPlibs.git
```

Comment:
no change needed for this PR, but this repo has been archived so we should probably use the NOAA version. https://github.com/NCAR/NCEPlibs. Could you make an issue to that effect?

Reply:
Sure thing -- see #2038.